talk/owt/sdk/base/stream.cc

```diff
@@ -82,15 +82,27 @@ Stream::Stream()
       renderer_impl_(nullptr),
       audio_renderer_impl_(nullptr),
       d3d9_renderer_impl_(nullptr),
+#ifdef OWT_ENABLE_QUIC
```

Comment:
You can simply set DataSourceInfo to unknown if QUIC is not enabled.

Reply:
This is to make sure all API/structures(struct StreamSourceInfo here) does not change if OWT_ENABLE_QUIC is not specified (current iOS and Linux SDK case) . Until all SDK upgrades to the version with QUIC support, I prefer this to be implemented like this.